integration_tests/tests/waf-regional.rs

```diff
@@ -0,0 +1,18 @@
+#![cfg(feature = "waf-regional")]
+
+extern crate rusoto_core;
+extern crate rusoto_waf_regional;
+
+use rusoto_waf_regional::{WAFRegional, WAFRegionalClient, ListRulesRequest};
+use rusoto_core::{DefaultCredentialsProvider, Region};
+use rusoto_core::default_tls_client;
+
+#[test]
+fn should_list_rules() {
+    let credentials = DefaultCredentialsProvider::new().unwrap();
+    let client = WAFRegionalClient::new(default_tls_client().unwrap(), credentials, Region::UsEast1);
+    let request = ListRulesRequest::default();
+
+    let result = client.list_rules(&request).unwrap();
```

Comment:
I don't entirely disagree, but I think the bigger issue here is that S3's integration tests are not isolated units with easily identifiable names. `test_all_the_things` makes it kind of tough to track down what's not working. 😉 

In this case, I'm not convinced adding an `expect(...)` message improves the test output in any meaningful way.

Current:
```
     Running target\debug\deps\waf_regional-74775e9c21b10e8e.exe

running 1 test
test should_list_rules ... FAILED

failures:

---- should_list_rules stdout ----
        thread 'should_list_rules' panicked at 'called `Result::unwrap()` on an `Err` value: Unknown("{\"__type\":\"AccessDeniedException\",\"Message\":\"User: arn:aws:iam::[redacted]:user/rusoto_integration is not authorized to perform: waf-regional:ListRules on resource: arn:aws:waf-regional:us-east-1:[redacted]:rule/*\"}")', src\libcore\result.rs:859
```

With expect:
```
     Running target\debug\deps\waf_regional-74775e9c21b10e8e.exe

running 1 test
test should_list_rules ... FAILED

failures:

---- should_list_rules stdout ----
        thread 'should_list_rules' panicked at 'waf regional list_rules call errored: Unknown("{\"__type\":\"AccessDeniedException\",\"Message\":\"User: arn:aws:iam::[redacted]:user/rusoto_integration is not authorized to perform: waf-regional:ListRules on resource: arn:aws:waf-regional:us-east-1:[redacted]:rule/*\"}")', src\libcore\result.rs:859
```

I'm biased against it though because it's a pain to go back through all these branches to modify that though. Let me know if you feel strongly about it.